routemodel/data_retrieval/get_weather.py

```diff
@@ -3,58 +3,82 @@
 sys.path.append(os.path.dirname(__file__))
 import requests
 import csv
-import json
 
 from config import WEATHER_API_KEY
+
 ONE_CALL_BASE = 'https://api.openweathermap.org/data/2.5/onecall?'
-PATH = os.path.join(os.path.dirname(__file__), '..', 'routes\ASC2021\ASC2021_draft.csv')
+PATH = os.path.join(os.path.dirname(__file__), '..', 'routes/ASC2021/ASC2021_draft.csv')
+
+
+def get_weather(lat, long, weather_api_result):
+    """
+    Calls OpenWeather API to get wind data and returns data in a list
+    @param lat: string containing latitude value
+    @param long: string containing longitude value
+    @param weather_api_result: list of lists containing weather data
+    @return: list of lists containing weather data
```

Comment:
(Not a change that you need to make, more for discussion) Do you think this would benefit from being a pd dataframe? Would save a lot of iteration, and accessing by indices which is harder to deal with?

Reply:
Hey so I wasn't sure if there was a reason that it was structured as a list when this file was initially created so that's why I left it as that. But I do agree with this it is difficult to understand because there are so many indices that have different fields.  I'll change it to a df.

Reply:
The only thing with this is because pd data frame don't allow for in-place append using a data frame does require more memory overall.